subprojects/language-java/src/integTest/groovy/org/gradle/java/compile/incremental/JavaClassChangeIncrementalCompilationIntegrationTest.groovy

```diff
@@ -409,9 +409,10 @@ class JavaClassChangeIncrementalCompilationIntegrationTest extends BaseJavaClass
         outputs.recompiledClasses('A', 'B')
     }
 
-    @NotYetImplemented
     @Issue("https://github.com/gradle/gradle/issues/20478")
-    def "can detect deletion of class used as type parameter"() {
+    def "can detect deletion of class used as class type parameter"() {
+        //executer.requireOwnGradleUserHomeDir()  //use when debugging to bypass cache
+
         source "interface Super <T> { T getResult(); }"
```

Comment:
Could we also add test just to be sure, when class have multiple parameters, e.g. `Super<Deleted, Deleted2>`?